src/notification.h

```diff
@@ -30,34 +30,44 @@ typedef struct _actions {
 } Actions;
 
 typedef struct _notification {
+        int id;
```

Comment:
I'd group this differently: `id` and `dbus_client` together and `appname`, `summary`, `body`, `category`, `urgency` in another group since they're the main notification attributes.

Reply:
Grouped it accordingly now.